opentelemetry-sdk/src/metrics/meter.rs

```diff
@@ -75,14 +75,18 @@ impl SdkMeter {
     {
         let validation_result = validate_instrument_config(builder.name.as_ref(), &builder.unit);
         if let Err(err) = validation_result {
-            global::handle_error(err);
+            otel_error!(
```

Comment:
otel_error!(
                name: "InstrumentCreationFailed", 
                meter_name = self.scope.name.as_ref(),
                instrument_name = builder.name.as_ref(),
                message = "Measurements from this instrument will be ignored."
                reason = fmt(err))

^ I prefer this version. Please see if this is better for end users.

Reply:
Error vs Warning - this is debatable. We need to find some agreement to follow throughout.
If entire telemetry flow is affected, then Error is apt.
for missing measurements from a single instrument - warn *might* be sufficient. This is good to be followed up separately anyway.